python/setup.py

```diff
@@ -23,6 +24,14 @@
     cc = os.environ.get("CC", "c++")
     os.environ['CC'] = 'ccache {}'.format(cc)
     print("Using 'ccache {}' as compiler".format(cc))
+
+    # This is very hacky but so is the entire setup.py buildsystem.
+    output=subprocess.check_output([cc, "--version"])
+    libs=['opmcommon', 'stdc++fs', 'boost_system']
```

Comment:
Do we still need `boost_system` - I thought that was purely transitive from `boost_filesystem`?

Reply:
still needed. i'll work on that later.